index.html

```diff
@@ -9025,6 +9039,267 @@ <h3><dfn>Take Element Screenshot</dfn></h3>
 </section> <!-- /Take Element Screenshot -->
 </section> <!-- /Screen capture -->
 
+<section>
+<h2>Print</h2>
+
+<p>The print functions are a mechanism to render the document to a
+ paginated format. It is returned to the <a>local end</a> as a Base64
+ encoded string containing a PDF representation of the paginated
+ document.
+
+<p class=note>
+For historical reasons, all dimensions relating to the printed output
+are specified in inches.
+
+<p>When required to <dfn>parse a page range</dfn> with
+arguments <var>pageRanges</var> and <var>totalPages</var>, an
+implementation must:
+
+<ol>
+  <li>Let <var>pages</var> be an empty <a>Set</a>
+
+  <li>For each <var>range</var> in <var>pageRanges</var>, run the
+  following steps:
+    <ol>
+      <li>If <var>range</var> is not either a <a>Number</a> or
+      a <a>String</a>, return <a>error</a> with <a>error
+      code</a> <a>invalid argument</a>.
+
+      <li><p>If <var>range</var> is a <a>Number</a>:
+        <ol>
+          <li>If <var>range</var> is not an integer or is less than 0,
+      return <a>error</a> with <a>error code</a> <a>invalid
+      argument</a>
```

Comment:
We also have a max limit, which we might never reach, but would it make sense to include a check here?

Reply:
What's the max limit?